docs/source/contributing/developing.adoc

```diff
@@ -126,12 +126,15 @@ $ dep ensure -v
 
 If your work requires a change to the dependencies, you need to update the Dep configuration.
 
-. Edit *_Gopkg.toml_* to change the dependencies as needed.
+. Refer to link:https://golang.github.io/dep/docs/daily-dep.html#updating-dependencies[dep document about updating dependencies]
 
-. Delete *_Gopkg.lock_* and re-create the vendor directory by running `make vendor`.
-Dep will recognize that there is no lock file and recalculate the required dependencies.
+. Test that everything still compiles with the new lock file in place by running `make clean && make`.
+
+[[add-dependency]]
+=== Adding a new Dependency
+If your work requires to add a new dependency, you need to update the Dep configuration.
 
-. Check-in the updated *_Gopkg.toml_* and *_Gopkg.lock_* files.
+. Refer to link:https://golang.github.io/dep/docs/daily-dep.html#adding-a-new-dependency[dep document about adding a dependency]
```

Comment:
Logically this change is fine, but I think it would be easier for the reader if we put the step here as it is just one step and then point to dep documentation for more details.

Reply:
@LalatenduMohanty problem is we can't put everything and putting minimum part here might confuse user so best to direct them to proper location which they can follow.